pkg/runtime/wfengine/backend.go

```diff
@@ -129,13 +129,7 @@ func (be *actorBackend) GetOrchestrationMetadata(ctx context.Context, id api.Ins
 	}
 
 	defer res.Close()
-	data, err := res.RawDataFull()
-	if err != nil {
-		return nil, fmt.Errorf("failed to read the internal actor response: %w", err)
-	}
-	if len(data) == 0 {
-		return nil, api.ErrInstanceNotFound
-	}
+	data := res.RawData()
```

Comment:
Same as previous comment, why are we now using RawData() and no longer error checking here?

Reply:
`DecodeInternalActorData` which is what this value is passed to internally uses `gob.NewDecoder` which accepts a stream.

`res.RawDataFull()` returns a byte slice, reading the stream fully in-memory first (internally, InvokeMethodResponse manages data as a stream).

`res.RawData()` just returns the stream.

Before, we were using `RawDataFull` to buffer the data in-memory, then internally in `DecodeInternalActorData` we were getting a readable stream from the data. So we were reading a stream, buffering in-memory, and then re-creating a stream from that.

Errors are now checked by `DecodeInternalActorData` while gob reads the stream.